src/CrawlTestLvl/private/world.cpp

```diff
@@ -36,7 +37,13 @@ CRAWL::WorldRule *World::initWorldRules() {
              {registerObject<CRAWL::DynamicWint>(), 1},
 
              {registerObject<Background>(), 1},
-             {registerObject<Day>(), 1}}}
+             {registerObject<Day>(), 1}}},
+        {2,
+            {
+                {registerObject<GroupObjBox>(),20}
```

Comment:
do you want to remove rest of this objects ?

because this line spawn only 20 GroupObjBox objects and remove another objects from world.